### ISSUE TITLE
Visiting /templates /starters /guides breaks routing

### ISSUE THREAD

**Issue by snehalbaghel** [maizzle/docs#126]
### Steps to reproduce

- Visit  https://maizzle.com/templates / https://maizzle.com/starters / https://maizzle.com/guides

OR

- Visit any page on in the `/docs` sub-route like: https://maizzle.com/docs/tailwindcss/
- Click on any of Templates / Starters / Guides links on the sidenav

### Current behaviour
Routing breaks for subsequent navigations in the sidenav

### Expected behaviour
Page content changes according to the route

**Comment by cossssmin** [comment]
Yeah, noticed that too, something’s bonkers in Gridsome. We’ll redesign the site and move to Nuxt eventually, there are more issues with Gridsome right now :(

**Comment by TamaraPK** [comment]
Hello, I just came across your Framework and it sounds great (already using Tailwind for web apps). I am facing the same issue as above. For now I am clicking enter on the address bar so that the browser actually loads the new path.
